src/openrct2/peep/Guest.cpp

```diff
@@ -4830,21 +4837,13 @@ void Guest::UpdateRideLeaveSpiralSlide()
     DestinationY = targetLoc.y;
 }
 
-/** rct2: 0x00981FE4 */
-static constexpr const uint8_t _MazeGetNewDirectionFromEdge[][4] = {
-    { 15, 7, 15, 7 },
-    { 11, 3, 11, 3 },
-    { 7, 15, 7, 15 },
-    { 3, 11, 3, 11 },
-};
-
-/** rct2: 0x00981FF4 */
-static constexpr const uint8_t _MazeCurrentDirectionToOpenHedge[][4] = {
-    { 1, 2, 14, 0 },
-    { 4, 5, 6, 2 },
-    { 6, 8, 9, 10 },
-    { 14, 10, 12, 13 },
-};
+/**
+ * Helper function to define maze pathfinding thresholds
+ */
+static constexpr uint16_t maze_forget_odds(double odds)
```

Comment:
```suggestion
static constexpr uint16_t MazeForgetOdds(double odds)
```

We're slowly converting everything to TitleCase, so better not adding any function in snake case.